src/PrestaShopBundle/Entity/Repository/RequestSqlRepository.php

```diff
@@ -112,10 +130,15 @@ public function getCountQueryBuilder(SearchCriteriaInterface $searchCriteria = n
      *
      * @param SearchCriteriaInterface $criteria
      *
+     * @deprecated deprecated since 1.7.8.0
+     * @see RequestSql::buildQueryBySearchCriteria
```

Comment:
I don't see where is `RequestSql` :thinking:

Reply:
Done ! Thanks